cirq/contrib/quirk/quirk_gate.py

```diff
@@ -116,65 +145,70 @@ def _gate_to_quirk_op(gate: ops.Gate) -> Optional[QuirkOp]:
     return None
 
 
-def x_to_known(gate: ops.XPowGate) -> Optional[QuirkOp]:
-    e = angle_to_exponent_key(gate.exponent)
-    if e is None:
-        return None
-    return QuirkOp('X' + e)
+def xyz_to_quirk_op(axis: str, gate: ops.EigenGate) -> QuirkOp:
```

Comment:
Optional: Signature of this function makes it appear that it is appropriate to use it for general EigenGates, not just Pauli-PowGates. It's easy to remedy this by taking global_shift and exponent as arguments in place of the gate.

Reply:
Acknowledged, but this is a private method not exposed to users.